buildsystem/scripts/copy_modules.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2017-2017 the openage authors. See copying.md for legal info.
+
+"""
+Copies the specified modules to the required directory.
+Used for packaging the python dependencies.
+"""
+
+import argparse
+import importlib
+import importlib.abc
+import os
+import shutil
+
+
+def copy_module(name, destination):
+    """Copy the importable module 'name' to the 'destination' directory"""
+    loader = importlib.find_loader(name)
+    if not isinstance(loader, importlib.abc.FileLoader):
+        print('Loader for module %s is not handled', name)
+        exit(1)
+
+    print('Copying "%s" to "%s"' % (name, destination))
+    filename = loader.get_filename(name)
+    if loader.is_package(name):
+        pkgdir, _ = os.path.split(filename)
+        shutil.copytree(pkgdir, os.path.join(destination, name))
+    else:
+        shutil.copy2(filename, destination)
+
+
+def main():
+    """ CLI entry point """
+    cli = argparse.ArgumentParser()
+    cli.add_argument("pymodule_name", nargs='+', help=(
+        "list of all modules that shall be copied"
+    ))
+    cli.add_argument("--dest_dir", required=True, help=(
+        "destination directory where modules will be copied"
+    ))
```

Comment:
If it's required it might be better to add `dest_dir` (without the `--`) as first argument, and then add `pymodule_names` as `nargs='+'` argument after that. Not important though, it's just unusual that `--arguments` are required.

Reply:
I thought having destination at the end would be more familiar (like `cp`). Removing the `--` from here seems to achieve that. I get this error when I try that: "TypeError: 'required' is an invalid argument for positionals". I have to remove `required=True` to fix that. Is that fine?
If not, I can switch the order. It might appear odd to a reader/user though.

Reply:
No `--` implies that its required. Imitating `cp` is a good idea, so we can keep the order, sure.